drltrace/CMakeLists.txt

```diff
@@ -70,7 +68,12 @@ use_DynamoRIO_extension(drltracelib drwrap_static)
 use_DynamoRIO_extension(drltracelib drx_static)
 use_DynamoRIO_extension(drltracelib drsyscall_static)
 use_DynamoRIO_extension(drltracelib drcovlib_static)
+use_DynamoRIO_extension(drltracelib droption)
 
+if (UNIX)
+  # to avoid conflicts with new declaration with "C" linkage in utils.h
```

Comment:
I have a different problem here. Got the following error:
In file included from /home/osboxes/drmemory_clean/build/dynamorio/ext/include/droption.h:40:0,
                 from /home/osboxes/drmemory_clean/src/drltrace/drltrace_options.h:33,
                 from /home/osboxes/drmemory_clean/src/drltrace/drltrace_frontend.cpp:44:
/usr/include/string.h:369:26: error: previous declaration of ‘const char* strcasestr(const char*, const char*)’ with ‘C++’ linkage
 extern "C++" const char *strcasestr (const char *__haystack,
                          ^
In file included from /home/osboxes/drmemory_clean/src/drltrace/drltrace_frontend.cpp:46:0:
/home/osboxes/drmemory_clean/src/common/utils.h:1006:49: error: conflicts with new declaration with ‘C’ linkage
 strcasestr(const char *text, const char *pattern);

Even if I don't use strcasestr at all.